submission.html

```diff
@@ -78,7 +85,7 @@ <h2>Dataset size</h2>
         <!-- begin prep section -->
         <hr class="whitespace">
         <div class="row column text-center">
-            <h2>Preparing metadata</h2>
+            <h2><a class="anchor" id="metadata"></a>Preparing the metadata</h2>
```

Comment:
In general, :+1:. Later, we might want to look into the little clickable anchor symbol to make it clear that this is something that people can share.